tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/ElasticsearchBoostedSearchTests.java

```diff
@@ -33,6 +21,18 @@
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
+import java.time.Clock;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+
+import static org.alfresco.elasticsearch.SearchQueryService.req;
```

Comment:
I think in this (if not all of our projects) import formatting is:
```
import static java.*
import static javax.*
<blank line>
import static all other imports
<blank line>
import java.*
import javax.*
<blank line>
import all other imports
```

Reply:
Done.